src/textual/app.py

```diff
@@ -128,13 +129,54 @@ class CssPathError(Exception):
 
 
 class _NullFile:
+    """A file-like where writes go nowhere."""
+
     def write(self, text: str) -> None:
         pass
 
     def flush(self) -> None:
         pass
 
 
+class _WriterThread(threading.Thread):
+    """A thread / file-like to do writes to stdout in the background."""
+
+    def __init__(self) -> None:
+        super().__init__(daemon=True)
+        self._queue: Queue[str | None] = Queue(32)
```

Comment:
What's the significance of `32` here? Might it be a good idea to make it a constant with a docstring to explain it, or at least a comment?

Reply:
Could use a constant.

Reply:
Its the number of enqueued writes before it blocks. Changed to a constant..